mne/stats/regression.py

```diff
@@ -246,6 +248,16 @@ def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
     """
 
     if isinstance(solver, string_types):
+        if solver == 'cholesky':
+            try:
+                from sklearn.linear_model import Ridge
+                def solver(X, y):
+                    return Ridge(fit_intercept=False, copy_X=False, alpha=0.,
+                                 solver='cholesky').fit(X, y.T).coef_
+            except ImportError:
+                warn("Can't import from scikit-learn. Falling back to 'pinv'.")
```

Comment:
actually I think this should be an error. If they ask for a method and it doesn't exist, error is more natural. (If we had an auto mode, or `cholesky` were the default, a warning might seem more appropriate.